crates/oxide/src/extractor/pre_processors/clojure.rs

```diff
@@ -5,6 +5,14 @@ use bstr::ByteSlice;
 #[derive(Debug, Default)]
 pub struct Clojure;
 
+#[inline]
+fn is_keyword_character(byte: u8) -> bool {
+    matches!(
+        byte,
+        b'+' | b'-' | b'/' | b'*' | b'_' | b'#' | b'.' | b':' | b'?'
+    ) | byte.is_ascii_alphanumeric()
+}
```

Comment:
```suggestion
/// This is meant to be a rough estimate of a valid ClojureScript keyword
///
/// This can be approximated by the following regex:
/// /::?[a-zA-Z0-9!#$%&*+./:<=>?_|-]+/
///
/// However, keywords are intended to be detected as utilities. Since the set
/// of valid characters in a utility (outside of arbitrary values) is smaller,
/// along with the fact that neither `[]` nor `()` are allowed in keywords we
/// can simplify this list quite a bit.
#[inline]
fn is_keyword_character(byte: u8) -> bool {
    return matches!(
        byte,
        b'!' | b'%' | b'*' | b'+' | b'-' | b'.' | b'/' | b':' | b'_'
    ) | byte.is_ascii_alphanumeric();
}
```

I updated this list to:
- Include `!` and `%` since they should be there
- Drop `#` and `?` as they can't be used in a named utility / variant today
- Order them in byte value order (not really needed just did it anyway)

Reply:
If this list looks good to you this should hopefully be the last change and can get this merged. I don't think I missed anything but lemme know if I did.